playbooks/azure/openshift-cluster/launch.yml

```diff
@@ -102,12 +103,9 @@
   # the cluster created in the previous steps to perform the e2e tests
   - name: persist the parameters file
```

Comment:
this is not just the parameters file anymore

Reply:
updated